web_sdk/test/api_conform_test.dart

```diff
@@ -19,7 +18,10 @@ const Set<String> _kObjectMembers = <String>{
 };
 
 CompilationUnit _parseAndCheckDart(String path) {
-  final FeatureSet analyzerFeatures = FeatureSet.fromEnableFlags(<String>['non-nullable']);
+  final FeatureSet analyzerFeatures = FeatureSet.fromEnableFlags2(
```

Comment:
Strictly not a nnbd change. Just moving away from the deprecated `FeatureSet.fromEnableFlags`.